massa-node/Cargo.toml

```diff
@@ -1,11 +1,12 @@
 [package]
-name = "massa-node"
-version = "0.1.0"
+name = "massa_node"
```

Comment:
This change and the one the client change the name of the binary also ?

Reply:
yes, the binary was renamed. I rollback my modification on it for now

Reply:
Ok yes thank you because it implies a lot for a lot of tooling etc

Reply:
Yes of course I had to check